src/pymovements/events/processing.py

```diff
@@ -41,9 +41,62 @@ class EventProcessor:
         List of event property names.
     """
 
-    def __init__(self, event_properties: str | list[str]):
+    def __init__(self, event_properties: str | tuple[str, dict] | list[str | tuple[str, dict]]):
         if isinstance(event_properties, str):
             event_properties = [event_properties]
+        elif isinstance(event_properties, tuple):
+            if len(event_properties) != 2:
+                raise ValueError('Tuple must have a length of 2.')
+            if not isinstance(event_properties[0], str):
+                raise TypeError(
+                    f"First item of tuple must be a string, but received {
```

Comment:
this line leads to a syntax error on python 3.9-3.11. the curly braces must be on the same line for these python versions.

Try to format it like you wrote it in `Dataset.compute_event_properties()` then this should work.